Lib/test/test_platform.py

```diff
@@ -219,6 +219,19 @@ def test_uname(self):
         self.assertEqual(res[-1], res.processor)
         self.assertEqual(len(res), 6)
 
+        if os.name == "posix":
+            uname = os.uname()
+            self.assertEqual(res.node, uname.nodename)
+            self.assertEqual(res.version, uname.version)
+            self.assertEqual(res.machine, uname.machine)
```

Comment:
This should be fairly safe, but could a core developer make sure by running this PR on the WASM buildbots, plus any other POSIX platforms which aren't covered by GitHub Actions?